Code/CMakeLists.txt

```diff
@@ -87,6 +88,9 @@ if(PAL_TRAIT_BUILD_HOST_TOOLS)
         BUILD_DEPENDENCIES
             PUBLIC
                 AZ::AzToolsFramework
+                Gem::AtomLyIntegration_CommonFeatures.Editor.Static
+                Gem::LmbrCentral.Editor
```

Comment:
A GEM_MODULE shouldn't be directly added as a dependency.
Instead the `LmbrCentral.API` target should be added as dependency as it is and INTERFACE target that provides the include directories for LmbrCentral.